src/target/source/codegen_cuda.h

```diff
@@ -109,6 +109,13 @@ class CodeGenCUDA final : public CodeGenC {
   // Op attribute map
   OpAttrMap<bool> op_need_warp_shuffle_ = Op::GetAttrMap<bool>("cuda.need_warp_shuffle");
 
+  // The name of the barrier array in shared memory
+  const std::string barrier_name_ = "barrier";
```

Comment:
This is definitely fine for now, but thinking ahead a little bit, will this cause issues if we try to have multiple Blocks in the same Thread Block Cluster, each with their own "barrier"? Depending on scoping it might be fine.

Reply:
Good point.  Scoping is kernel scope.  Let's leave this for now and address problems as they come up.